src/components/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -0,0 +1,553 @@
+<template>
```

Comment:
Is there a good way to share this fairly large component. It looks like the only difference from the existing page is the use of `WwwPage`.